bot.py

```diff
@@ -48,6 +48,13 @@ def basename(cards):
 def unaccent(s):
   return ''.join((c for c in unicodedata.normalize('NFD', s) if unicodedata.category(c) != 'Mn'))
 
+def prettify_mana_cost(cost):
+  cost = re.sub(r'\{([0-9])\}', r':0\g<1>:', cost)
```

Comment:
Because the backref is immediately followed by a colon, it was complaining about an Invalid Group Reference.
The verbose \g<1> fixed that.